DotNetHowToAutocomplete/DotNetHowToAutocomplete/App_Start/BundleConfig.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Web;
+using System.Web.Optimization;
+
+namespace AutocompleteTutorial
+{
+    public class BundleConfig
+    {
+        // For more information on bundling, visit https://go.microsoft.com/fwlink/?LinkId=301862
+        public static void RegisterBundles(BundleCollection bundles)
```

Comment:
How much of this is stuff we need, and how much are dependencies that VS throws in just in case?

Reply:
Most of this is from File->New->ASP.NET MVC project.  I guess I can clear stuff out, but was thinking it was harmless since it is standard in an MVC project.